server.php

```diff
@@ -1,21 +0,0 @@
-<?php
```

Comment:
I don't know why this file was deleted. Please explain the purpose of deleting this file. Upon then, will I reopen this PR and resolve any issue for merging.